src/app/component/MissionCard.jsx

```diff
@@ -56,10 +49,12 @@ const MissionCardContent = ({ contentItems, classes }) => (
  * @component
  */
 const MissionCard = withStyles(styles)(({ mission, children, classes, ...rest }) => {
+  console.log(mission);
```

Comment:
Can we remove the console log if its only used for debugging?

Reply:
Missed that one. Thanks!